src/__tests__/formatPkg.test.js

```diff
@@ -377,3 +377,67 @@ describe('test getRepositoryInfo', () => {
     expect(getRepositoryInfo('aaaaaaaa')).toBe(null);
   });
 });
+
+describe('alternative names', () => {
+  test('name not yet ending in .js', () => {
+    const original = {
+      name: 'places',
+      lastPublisher: { name: 'unknown' },
+    };
+    expect(formatPkg(original)._searchInternal.alternativeNames)
```

Comment:
If you are specifically testing `getAlternativeNames` you could use it directly, that way the test would be clear about the expectation. You have already have snapshot in place to be sure that this function is already called in the main method.

Reply:
good point, but would leave it for a major reorganise, since these functions aren't exported :)